sweepbatcher/sweep_batcher.go

```diff
@@ -324,6 +331,9 @@ func (b *Batcher) handleSweep(ctx context.Context, sweep *sweep,
 					"accepted by batch %d", sweep.swapHash[:6],
 					batch.id)
 			}
+
+			// The sweep was updated in the batch, our job is done.
+			return nil
```

Comment:
Isn't this a bug that should be addressed in a separate commit?
Is it right that without this change, we might add the sweep to another batch in the loop below even though it already existed in another batch?

Reply:
I think it is possible. I'll try to create a unit test to trigger the bug condition. I think it worth doing it in a separate PR.

Reply:
Yeah make sense. IIUC it might be fine if a sweep exists in multiple batches as a sweep spend would notify all batches, cc @GeorgeTsagk